internal/cmd/beta/network-area/route/describe/describe.go

```diff
@@ -134,9 +135,18 @@ func outputResult(p *print.Printer, outputFormat string, route *iaas.Route) erro
 		table := tables.NewTable()
 		table.AddRow("ID", *route.RouteId)
 		table.AddSeparator()
-		table.AddRow("Prefix", *route.Prefix)
+		table.AddRow("PREFIX", *route.Prefix)
 		table.AddSeparator()
-		table.AddRow("Nexthop", *route.Nexthop)
+		table.AddRow("NEXTHOP", *route.Nexthop)
+		table.AddSeparator()
+		if route.Labels != nil && len(*route.Labels) > 0 {
+			labels := []string{}
+			for key, value := range *route.Labels {
+				labels = append(labels, fmt.Sprintf("%s: %s", key, value))
+			}
+			table.AddSeparator()
+			table.AddRow("LABELS", strings.Join(labels, "\n"))
+		}
```

Comment:
The separation from line 141 is not needed, right? As if the network has labels, it will be added in line 147 already

Reply:
Yes, you are right :+1: 
Line removed